code/code/misc/extern.h

```diff
@@ -349,7 +349,7 @@ extern int mapSpellnumToFile(spellNumT);
 extern spellNumT mapFileToSpellnum(int);
 extern bool applyTypeShouldBeSpellnum(applyTypeT);
 extern bool has_key(TBeing *ch, int key);
-extern int bogusAccountName(const char *arg);
+extern int bogusAccountName(const sstring arg);
```

Comment:
```cpp
extern bool bogusAccountName(const sstring& arg);
```

also I'd stick the extern into `client.cc` and remove from here.

Reply:
the less in extern.h the better for sure.